tests/sensors/test_sql_sensor.py

```diff
@@ -103,6 +103,18 @@ def test_sql_sensor_postgres_poke(self, mock_hook):
         mock_get_records.return_value = []
         self.assertFalse(t.poke(None))
 
+        mock_get_records.return_value = [['None']]
```

Comment:
This is when it returns the string `"None"` ritght?

Shouldn't we also have a case where we test the behaviour with `[[None]]` as the return?

Reply:
I guess `mock_get_records.return_value = []` would check exactly that issue. But I added another case to test for `[[None]]` explicitly